templates/events/members.html.twig

```diff
@@ -160,34 +176,24 @@
                 </h2>
                 <div class="committee__members__nav">
                     <div>
-                        <form method="post" action="{{ path("app_#{event.type}_contact_members", {'slug': event.slug}) }}">
+                        <form method="post" action="{{ path("app_#{event.type}_contact_members", { slug: event.slug }) }}">
```

Comment:
Tip: to use the constant here and avoid the interpolation at the same time you can use the filter [`format`](https://twig.symfony.com/doc/2.x/filters/format.html) which acts like `sprintf` ;).

Reply:
Thanks, but it seems to me not applicable here in `path()`.

Reply:
Done